### ISSUE TITLE
Sync groups

### ISSUE THREAD

**Issue by EionRobb** [EionRobb/purple-hangouts-attempt2#87]
**[Original report](https://bitbucket.org/EionRobb/purple-hangouts/issue/87) by Ross Hadden (Bitbucket: [rosshadden](https://bitbucket.org/rosshadden), GitHub: [rosshadden](https://github.com/rosshadden)).**

----------------------------------------

Is it feasible to sync groups with Google's api somehow?  To be clear, I am referring to buddy list groups, not group chats.

I know that Hangouts itself has no grouping support (at least in the UI), but the groups added with the good ol' Google Talk xmpp accounts do support this.

Basically I noticed that if I move contacts to groups in my buddy list, the changes are not present on my other computers.  I used to sync the buddylist xml file, but stopped years ago when I realized that the xmpp groups were already syncing on their own.
